src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilationBuilder.cs

```diff
@@ -28,12 +29,14 @@ public sealed class ReadyToRunCodegenCompilationBuilder : CompilationBuilder
         private KeyValuePair<string, string>[] _ryujitOptions = Array.Empty<KeyValuePair<string, string>>();
         private ILProvider _ilProvider = new ReadyToRunILProvider();
 
-        public ReadyToRunCodegenCompilationBuilder(CompilerTypeSystemContext context, CompilationModuleGroup group, string inputFilePath, bool ibcTuning, bool resilient)
+        public ReadyToRunCodegenCompilationBuilder(CompilerTypeSystemContext context, CompilationModuleGroup group,
```

Comment:
This is a [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern) that (besides other things) accomplishes that we don't end up with a constructor with 50 parameters that one needs to specify before getting something usable back.

Can you follow this pattern? (Add `UseJitPath` method.)

Reply:
The pattern seems somewhat broken since we still pass a number of arguments to the constructor (ibcTuning, resilient, etc.), but yes, I can follow it.

Reply:
> The pattern seems somewhat broken since we still pass a number of arguments to the constructor (ibcTuning, resilient, etc.)

Yes, that my fault for not spotting it in David's huge pull requests a couple weeks ago. I was focused on other things. I'll fix those when this merges.